tests/app/utilities/test_schema.py

```diff
@@ -155,6 +155,20 @@ def test_load_schema_from_url_200():
     assert cache_info.hits == 0
 
 
+@responses.activate
+def test_load_schema_from_url_connection_error():
+    load_schema_from_url.cache_clear()
+
+    responses.add(responses.GET, "", json={"error": "not found"}, status=0)
```

Comment:
Is this line needed?.

Reply:
Yeah, works fine without it so I will remove it, thanks.